catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/ResourceOperations.java

```diff
@@ -552,139 +548,100 @@ protected ResourceInfo getResourceInfo(ResourceRequest resourceRequest, String s
-                                    resourceUri.getSchemeSpecificPart(),
-                                    null);
-                        } catch (URISyntaxException e) {
-                            throw new ResourceNotFoundException(
-                                    "Could not resolve URI by doing a URI based query: " + value);
-                        }
-                    }
-
-                    Query propertyEqualToUriQuery =
-                            createPropertyIsEqualToQuery(Metacard.RESOURCE_URI,
-                                    resourceUri.toString());
-
-                    // if isEnterprise, go out and obtain the actual source
-                    // where the product's metacard is stored.
-                    QueryRequest queryRequest = new QueryRequestImpl(anyTag(propertyEqualToUriQuery,
-                            site,
-                            isEnterprise),
-                            isEnterprise,
-                            Collections.singletonList(site == null ? this.getId() : site),
-                            resourceRequest.getProperties());
-
-                    QueryResponse queryResponse = queryOperations.query(queryRequest,
-                            null,
-                            true,
-                            fanoutEnabled);
-                    if (!queryResponse.getResults()
-                            .isEmpty()) {
-                        metacard = queryResponse.getResults()
-                                .get(0)
-                                .getMetacard();
-                        federatedSite.append(metacard.getSourceId());
-                        LOGGER.debug("Trying to lookup resource URI {} for metacardId: {}",
-                                resourceUri,
-                                resourceUri);
-
-                        if (!requestProperties.containsKey(Metacard.ID)) {
-                            requestProperties.put(Metacard.ID, metacard.getId());
-                        }
-                        if (!requestProperties.containsKey(Metacard.RESOURCE_URI)) {
-                            requestProperties.put(Metacard.RESOURCE_URI, metacard.getResourceURI());
-                        }
-                    } else {
-                        throw new ResourceNotFoundException(
-                                "Could not resolve source id for URI by doing a URI based query: "
-                                        + resourceUri);
-                    }
-                } else {
-                    throw new ResourceNotSupportedException(
-                            "The GetResourceRequest with attribute value of class '"
-                                    + value.getClass()
-                                    + "' is not supported by this instance of the CatalogFramework.");
-                }
+                URI resourceUri = (URI) value;
+                query = createPropertyIsEqualToQuery(Metacard.RESOURCE_URI,
```

Comment:
If this is a derived resource this query will return 0 results and then it will fail to retrieve the resource.  The previous logic assumed anytime a `fragment` occurred in the URI, the resource was derived but that was a poor assumption.  We either need to be able to query the derived resource attribute or prune the URI to match like it was before.